<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderWindowUISingleInheritance</class>
 <widget class="QMainWindow" name="RenderWindowUISingleInheritance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>.mat Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QVTKWidget" name="qvtkWidget">
          <property name="geometry">
            <rect>
              <x>10</x>
              <y>20</y>
              <width>700</width>
              <height>700</height>
            </rect>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QTextEdit" name="logEdit">
          <property name="geometry">
            <rect>
              <x>10</x>
              <y>20</y>
              <width>20</width>
              <height>20</height>
            </rect>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="label_2">
          <property name="text">
            <string>.mat files</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="matEdit"/>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
            <widget class="QLabel" name="label_3">
              <property name="text">
                <string>Variable Name</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLabel" name="label_4">
              <property name="text">
                <string>phi</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLabel" name="label_5">
              <property name="text">
                <string>theta</string>
              </property>
            </widget>
          </item>        
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
            <widget class="QLineEdit" name="matVarEdit"/>
          </item>
          <item>
            <widget class="QLineEdit" name="generalEdit_1"/>
          </item>
          <item>
            <widget class="QLineEdit" name="generalEdit_2"/>
          </item>        
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <widget class="QPushButton" name="startButton">
              <property name="text">
                <string>Run</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="importButton">
              <property name="text">
                <string>Open files</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>

 </resources>
 <connections/>
</ui>
